### PR TITLE
SAK-46032 Change Shared rubrics to Public rubrics to clarify function

### PR DESCRIPTION
**Pull request by jkozar2**
This PR alters Shared Rubrics to become Public Rubrics to make it clearer how sharing works; we thought Public better reflects the wide access given a rubric when it's shared.

### CODE REVIEW

**Comment by jkozar2** [comment]
My apologies...I was trying to resolve a merge conflict and that seems to have merged Master into this somehow. I'm working on trying to fix it up now.

**Comment by ottenhoff** [comment]
Merging master into your branch is fine ..... the diff still looks normal

File: rubrics/api/src/main/bundle/rubrics.properties [hunk 1]
```diff
@@ -64,3 +64,5 @@ still, however, see their rubric evaluations in tools like Assignments, Gradeboo
 Quizzes. You DO NOT need to unlock this tool for students to see their rubric evaluations.
 adjust_scores_warning=Deselecting the 'Adjust individual student scores' option will reset \
 existing student rubric scores to the default point value for selected ratings. Are you sure?
+public_rubrics_title=Publicly Shared Rubrics
+public_rubrics_info=Rubrics in this section were created by instructors and shared publicly with all Isidore users. You can share a rubric publicly by clicking on the "Globe" icon next to the rubric you would like to share. To use one of these rubrics in your Isidore site, expand the "Public Rubrics" section and click on the "Copy" icon located in the "Actions" column.
```
Comment (adrianfish) [inline rubrics/api/src/main/bundle/rubrics.properties:68]: Isidore will need to be made generic.

Comment (jkozar2) [reply rubrics/api/src/main/bundle/rubrics.properties]: Fix just committed!

**Review by ern — APPROVED** [review]

**Pull request merged**